test/server/routes/insertTaskRoute.js

```diff
@@ -46,7 +46,7 @@ test('Testing for api/v1/tasks/new route', (t) => {
   selectUserScrumProj()
     .then((result) => {
       newTask.scrum_id = result.rows[0].scrum_id;
-      newTask.assigned_to = result.rows[0].user_id;
+      newTask.assigned_to = result.rows[0].full_name;
```

Comment:
shouldn't this be user_name?

Reply:
Why user_name? I think full_name is more appropriate in this case. User_names are only for login. What do you think?